app/src/main/java/com/lagradost/cloudstream3/utils/VideoDownloadManager.kt

```diff
@@ -1733,7 +1738,37 @@ object VideoDownloadManager {
         }
     }
 
-    fun deleteFileAndUpdateSettings(context: Context, id: Int): Boolean {
+    fun deleteFilesAndUpdateSettings(
+        context: Context,
+        ids: List<Int>,
```

Comment:
I would much rather pass in a set to indicate that you cant delete the same file twice or other fuckery.

Reply:
Done